javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,76 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+    convertToCurrency(currency){
+      const newPrice = 0
+      console.log(this.price);
+      if (currency == 'dollars') {
+        let newPrice = this.price / 6.27
+      // convertedPrice.innerText = this.name +' =' + newPrice.toFixed(1) +' $'
+        return console.log(newPrice);
+      } else if (currency == 'euro') {
+        let newPrice = this.price / 7.44
+        //convertedPrice.innerText = this.name + '= '+newPrice.toFixed(1) +' Euro'
+        return console.log(newPrice);
+      }    
+    }
+  }
+ class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      this.products.push(product)
+    }
+  
+    removeProduct(product) {
+      // Implement functionality here
+      const removedProduct = this.products.splice(item => item.name != product)
+      this.products = removedProduct
+    }
+  
+    searchProduct(productName) {
+      // Implement functionality here
+      const findingProducts = this.products.find(item => {return item.name.toLowerCase() == productName.toLowerCase() })
+      this.products = searchedProduct;
```

Comment:
what is `searchedProduct`? You mean `findingProducts`? Without running the code, i have a feeling it won't work.